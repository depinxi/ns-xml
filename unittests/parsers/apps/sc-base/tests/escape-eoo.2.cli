sc "Escaping" "end-of-options" "\--" --argument-b="Again" -ae