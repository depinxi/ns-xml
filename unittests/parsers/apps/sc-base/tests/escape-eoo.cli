"Escaping" "end-of-options" \\-- -a