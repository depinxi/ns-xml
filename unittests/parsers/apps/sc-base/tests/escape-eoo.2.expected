CLI: "sc", "Escaping", "end-of-options", "\--", "--argument-b=Again", "-ae"
Value count: 3
Values: "Escaping", "end-of-options", "--"
Error count: 0
Subcommand: sub
switchA=False
argumentB=Again
sub_argumentD=
sub_switchE=True
sub_switchA2=True
